0-bootstrap/main.tf

```
@@ -125,7 +128,7 @@ module "cloudbuild_bootstrap" {
   terraform_version_sha256sum = "4a52886e019b4fdad2439da5ff43388bbcc6cce9784fde32c53dcd0e28ca9957"
 
   # Remove after github.com/terraform-google-modules/terraform-google-bootstrap/issues/160
-  depends_on = [module.seed_bootstrap]
+  depends_on = [module.seed_bootstrap, module.google_groups.required_groups]
```

```suggestion
  depends_on = [module.seed_bootstrap, module.google_groups[0].required_groups]
```

but I thin it could be only

```suggestion
  depends_on = [module.seed_bootstrap, module.google_groups]
```